server/pulp/server/webservices/views/serializers/link.py

```diff
@@ -0,0 +1,59 @@
+"""
+Generation of link objects for REST object serialization.
+
+link object:
+{
+    "_href": <uri path to resource or collection>
+}
+"""
+
+import copy
+
+from pulp.server.webservices import http
+
+
+_LINK_OBJ_SKEL = {
```

Comment:
Please turn this into a factory method that can be called to return a this dict. That would be better than what we have currently which relies on anyone who uses this to remember to call `copy.copy()` or else leave a dirty state in this module level attribute. Once changed, any code that touched this object should use the factory and don't need to call `copy.copy()` all over. Also this new factory needs test coverage too.